wiki/Tournaments/GBC/GBC_2023_Autumn/en.md

```diff
@@ -33,27 +33,63 @@ The **GB Cup 2023 Autumn** (***GBC 2023 Autumn***) is a 3v3 captain draft–base
 
 ## Prizes
 
-| Placing | Prizes |
+### QQ Division
+
+| Team Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, ¥150 CNY |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | KanonBot profile badge, ¥150 CNY |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | KanonBot profile badge, ¥120 CNY |
+| *4th place* | ¥90 CNY |
+| *5–6th place* | ¥75 CNY |
+| *7–8th place* | ¥60 CNY |
+| *9–12th place* | ¥45 CNY |
+| *13–16st place* | ¥30 CNY |
+| Every Winning Match in Regular Stage | ¥50 CNY |
```

Comment:
what does "Every Winning Match in Regular Stage" this mean? it doesn't make sense rn
it should also be in sentence case

Reply:
```suggestion
| *For every match won in the regular stage* | ¥50 CNY |
```